cfgmgr/intfmgr.cpp

```diff
@@ -287,22 +301,149 @@ void IntfMgr::addHostSubIntf(const string&intf, const string &subIntf, const str
     EXEC_WITH_ERROR_THROW(cmd.str(), res);
 }
 
-void IntfMgr::setHostSubIntfMtu(const string &subIntf, const string &mtu)
+
+std::string IntfMgr::getPortAdminStatus(const string &alias)
+{
+    Table *portTable;
+    if (!alias.compare(0, strlen("Eth"), "Eth"))
+    {
+        portTable = &m_statePortTable;
+    }
+    else if (!alias.compare(0, strlen("Po"), "Po"))
+    {
+        portTable = &m_appLagTable;
+    }
+    else
+    {
+        return "down";
+    }
+
+    string admin = "down";
+    vector<FieldValueTuple> temp;
+    portTable->get(alias, temp);
+
+    for (auto idx : temp)
+    {
+        const auto &field = fvField(idx);
+        const auto &value = fvValue(idx);
+        if (field == "admin_status")
+        {
+            admin = value;
+        }
+    }
+    return admin;
+}
+
+std::string IntfMgr::getPortMtu(const string &alias)
```

Comment:
Replace port with intf here as well in the func. name.

Reply:
sure